src/core/core_utils.js

```diff
@@ -287,19 +287,26 @@ function _collectJS(entry, xref, list, parents) {
 
 function collectActions(xref, dict, eventType) {
   const actions = Object.create(null);
-  if (dict.has("AA")) {
-    const additionalActions = dict.get("AA");
-    for (const key of additionalActions.getKeys()) {
-      const action = eventType[key];
-      if (!action) {
-        continue;
-      }
-      const actionDict = additionalActions.getRaw(key);
-      const parents = new RefSet();
-      const list = [];
-      _collectJS(actionDict, xref, list, parents);
-      if (list.length > 0) {
-        actions[action] = list;
+  const additionalActionsDicts = getInheritableProperty({
+    dict,
+    key: "AA",
+    stopWhenFound: false,
+  });
+  if (additionalActionsDicts) {
+    for (let i = additionalActionsDicts.length - 1; i >= 0; i--) {
```

Comment:
I'm assuming that there's a particular reason for iterating through this array in *reverse* order, so small comment mentioning why would probably not hurt here (to avoid something changing that later thinking it won't matter).